website/docs/Installation.md

```diff
@@ -1,5 +1,14 @@
 # Installation
 
+## Setup Virtual Environment
+
+We recommend using a virtual environment to install AutoGen. This will ensure that the dependencies for AutoGen are isolated from the rest of your system.
```

Comment:
```suggestion
When not using a docker container, we recommend using a virtual environment to install AutoGen. This will ensure that the dependencies for AutoGen are isolated from the rest of your system.
```